benchmarks/maxtext_trillium_model_configs.py

```diff
@@ -320,6 +359,45 @@ class MaxTextModel:
     ),
 )
 
+llama2_70b_4096_pw_long_run = MaxTextModel(
+    model_name="llama2-70b-4096-pw-lr",
+    model_type="llama2-70b",
+    tuning_params={
+        "per_device_batch_size": 4,
+        "ici_fsdp_parallelism": 1,
+        "ici_fsdp_transpose_parallelism": -1,
+        "ici_tensor_parallelism": 1,
+        "remat_policy": "full",
+        "max_target_length": 4096,
+        "attention": "flash",
+        "gcs_metrics": True,
+        "use_iota_embed": True,
+        "dataset_path": "gs://max-datasets-rogue",
+        "dataset_type": "synthetic",
+        "reuse_example_batch": 1,
+        "profiler": "xplane",
+        "sa_block_q": 1024,
+        "sa_block_q_dkv": 2048,
+        "sa_block_q_dq": 2048,
+
+        # Additional tuning params for pathways long running test.
```

Comment:
can we move pathways long running test related configurations like ``` "metrics_file": "metrics.txt",
        "goodput_upload_interval_seconds": 30,
        "enable_pathways_goodput": True,
        "enable_checkpoint_cloud_logger": True,
        "enable_single_controller": True, ``` in maxtext_xpk_runner? like new function `xpk-command-for-pathway`, [maxtext_trillium_model_configs.py](https://github.com/AI-Hypercomputer/maxtext/pull/1094/files#diff-5c68d18bef03e76cee74107f324af52adf487414f181baf577cdc1aad0618e8f) is supposed to have different benchmarks. In that case, it will be easy for pathway to run different benchmarks. What do you think?

Reply:
These configs are actually for the specific test we're running (long running tests). I think keeping it as this config would make more sense because we may have different tests that are just for perf, or non goodput related, etc. I wouldn't want to bifurcate the code too much, it's much easier for us to just add a specialized config for Pathways and flip on the specific flags for that test I think.